operator/src/main/java/oracle/kubernetes/operator/WebhookLiveness.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Date;
+
+import oracle.kubernetes.operator.logging.LoggingFacade;
+import oracle.kubernetes.operator.logging.LoggingFactory;
+import oracle.kubernetes.operator.logging.MessageKeys;
+
+/**
+ * This task maintains the "liveness" indicator so that Kubernetes knows the Webhook is still
+ * alive.
+ */
+public class WebhookLiveness implements Runnable {
```

Comment:
This is another copy... Similarly, there could be one Liveness class that takes constructor arguments for the differences.

Reply:
The `OperatorLiveness` class takes MainDelegate as a parameter and creates the liveness file in `OperatorHome` returned by the delegate which defaults to `/operator`. Do we want to create the webhook liveness file under '/operator' or '/webhook' or should we rename the `OperatorHome` to something like `DeploymentHome` which can be used by both the operator and the webhook and defaults to something like `/deployments`?

Reply:
Yes, some common path. /probes ?

Reply:
Changed.